docs/rfc/rfc-026-p2p-bad-peers-checktx.md

```diff
@@ -0,0 +1,375 @@
+# RFC 26: Banning peers based on ResponseCheckTx
+
+## Changelog
+- Nov 4, 2022: Initial draft (jmalicevic)
+- Nov 8, 2022: Updated draft (jmalicevic)
+
+## Abstract
+
+Before adding a transaction to its mempool, a node runs `CheckTx` on it. `CheckTx` returns a non-zero code
+in case the transaction fails an application specific check. 
```

Comment:
Slight tangent, but it might be helpful for the reader to clarify that transactions can be added through the RPC from an external client or via another peer and that for this RFC we only consider the case where it is received from another peer.